exonum-java-binding/package_app.py

```diff
@@ -0,0 +1,74 @@
+import os
+import sys
+import subprocess
+from shutil import copy
+from pathlib import Path
+from tests_profile import TestsProfile
+
+
+def set_packaging_rustflags():
+    # No need on Windows
+    if sys.platform == 'win32':
+        return
+
+    if sys.platform == 'linux2':
+        prefix = "\$ORIGIN"
+    elif sys.platform == 'darwin':
+        prefix = "@loader_path"
+    else:
+        raise Exception(f"This OS ({sys.platform}) is not supported")
+
+    lib_path_relative_to_exe = prefix + "/lib/native"
+    rustflags = f"-C link-arg=-Wl,-rpath,{lib_path_relative_to_exe} -C link-arg=-Wl,-rpath,${prefix}"
+    print(f"Setting new RUSTFLAGS={rustflags}")
+    os.environ["RUSTFLAGS"] = rustflags
+
+
+def clear_cargo(manifest_dir):
+    manifest_path = os.path.join(manifest_dir, "Cargo.toml")
+    subprocess.run(["cargo", "clean", "--manifest-path", manifest_path], check=True)
+
+
+if __name__ == '__main__':
+    tests_profile = TestsProfile()
```

Comment:
As tests profile sets different rust flags, overwritten here, shall it be created here?

Reply:
Yes, probably we can implement `PackageProfile` subclass, I also noticed that.